gapic/src/main/com/google/gapid/views/ShaderView.java

```diff
@@ -123,7 +123,7 @@ public static class ShaderWidget extends Composite {
     private final TabFolder tabFolder;
     private final Composite sourceContainer;
     private final Group statGroup;
-    private final TableViewer statTable;
+    protected final TableViewer statTable;
```

Comment:
Why did this need to change?

Reply:
It's referenced from within an inner class. Inner classes are a language thing, but not a JVM thing, so the compiler has to add a getter function so the inner class can get access to the field. Making it protected avoids that.